server/app/playbook.go

```diff
@@ -14,65 +14,83 @@ import (
+	MessageOnJoin                        string                 `json:"message_on_join" export:"message_on_join"`
+	MessageOnJoinEnabled                 bool                   `json:"message_on_join_enabled" export:"message_on_join_enabled"`
+	RetrospectiveReminderIntervalSeconds int64                  `json:"retrospective_reminder_interval_seconds" export:"retrospective_reminder_interval_seconds"`
+	RetrospectiveTemplate                string                 `json:"retrospective_template" export:"retrospective_template"`
+	RetrospectiveEnabled                 bool                   `json:"retrospective_enabled" export:"retrospective_enabled"`
+	WebhookOnStatusUpdateURLs            []string               `json:"webhook_on_status_update_urls" export:"-"`
+	WebhookOnStatusUpdateEnabled         bool                   `json:"webhook_on_status_update_enabled" export:"-"`
+	SignalAnyKeywords                    []string               `json:"signal_any_keywords" export:"signal_any_keywords"`
+	SignalAnyKeywordsEnabled             bool                   `json:"signal_any_keywords_enabled" export:"signal_any_keywords_enabled"`
+	CategorizeChannelEnabled             bool                   `json:"categorize_channel_enabled" export:"categorize_channel_enabled"`
+	CategoryName                         string                 `json:"category_name" export:"category_name"`
+	RunSummaryTemplateEnabled            bool                   `json:"run_summary_template_enabled" export:"run_summary_template_enabled"`
+	RunSummaryTemplate                   string                 `json:"run_summary_template" export:"run_summary_template"`
+	ChannelNameTemplate                  string                 `json:"channel_name_template" export:"channel_name_template"`
+	DefaultPlaybookAdminRole             string                 `json:"default_playbook_admin_role" export:"-"`
+	DefaultPlaybookMemberRole            string                 `json:"default_playbook_member_role" export:"-"`
+	DefaultRunAdminRole                  string                 `json:"default_run_admin_role" export:"-"`
+	DefaultRunMemberRole                 string                 `json:"default_run_member_role" export:"-"`
+	Metrics                              []PlaybookMetricConfig `json:"metrics" export:"-"`
 }
 
 const (
 	PlaybookRoleMember = "playbook_member"
 	PlaybookRoleAdmin  = "playbook_admin"
 )
 
+const (
+	MetricTypeDuration = "metric_duration"
+	MetricTypeCurrency = "metric_currency"
+	MetricTypeInteger  = "metric_integer"
+)
+
+const MaxMetricsPerPlaybook = 4
+
 type PlaybookMember struct {
 	UserID      string   `json:"user_id"`
 	Roles       []string `json:"roles"`
 	SchemeRoles []string `json:"scheme_roles"`
 }
 
+type PlaybookMetricConfig struct {
+	ID          string `json:"id"`
+	PlaybookID  string `json:"playbook_id"`
+	Title       string `json:"title"`
+	Description string `json:"description"`
+	Type        string `json:"type"`
+	Target      int64  `json:"Target"`
```

Comment:
This caused a bug in the frontend:

```suggestion
	Target      int64  `json:"target"`
```

Reply:
Oh, I missed that : )) Fixed.